src/components/AuthProvider/AuthProvider.tsx

```
@@ -3,10 +3,12 @@ import {
   DynamicContextProvider,
   DynamicWidget,
 } from '@dynamic-labs/sdk-react-core';
-import settings from '@base/settings.json';
-import { isClient, isProd } from '@utils/common';
+import { isClient } from '@utils/common';
 import { createContext, useMemo, useState } from 'react';
 
+const DYNAMIC_ENV_ID =
+  import.meta.env?.PUBLIC_DYNAMIC_ENVIRONMENT_ID || 'UNAVAILABLE';
```

You can throw if missing since its critical